services/gitlab/gitlab-tag.service.js

```diff
@@ -0,0 +1,131 @@
+import Joi from 'joi'
+import { version as versionColor } from '../color-formatters.js'
+import { optionalUrl } from '../validators.js'
+import { latest } from '../version.js'
+import { addv } from '../text-formatters.js'
+import { NotFound } from '../index.js'
+import GitLabBase from './gitlab-base.js'
+
+const schema = Joi.array().items(
+  Joi.object({
+    name: Joi.string().required(),
+  })
+)
+
+const queryParamSchema = Joi.object({
+  gitlab_url: optionalUrl,
+  include_prereleases: Joi.equal(''),
+  sort: Joi.string().valid('date', 'semver').default('date'),
+}).required()
+
+export default class GitlabTag extends GitLabBase {
+  static category = 'version'
+
+  static route = {
+    base: 'gitlab/v/tag',
```

Comment:
on the recurring theme of consistency, both with github and at large, i've updated the route to match the github tag badge route. however, i realize i'm not completely convinced this aligns with the `service/noun/parameters` badge route pattern enumerated in our docs.

we've also got other github routes that have the `v` and `noun` order reversed, e.g. `github/v/tag/...` vs `github/package-json/v/...`

i think this is probably the best bet to provide the highest degree of consistency, but also feel like maybe the docs could use more clarity if this is a common exception to the main rule?

Reply:
Agree with your decision here :+1: